website/source/intro/getting-started/agent.html.md

```diff
@@ -9,6 +9,8 @@ description: >
   as data loss is inevitable in a failure scenario.
 ---
 
+**Note: This guide is temporarily being managed in a different repo. Please contact kait@hashicorp.com if you have any questions. 
```

Comment:
I actually think we can remove these. I could see people missing them if they went to update the guide. Prefer to have @kaitlincarter-hc just watch PRs closely.

Reply:
Files have been left in the repo but the prose has been removed.